### DISCUSSION TITLE
Vercel application installed to GitHub, but cant see/import any repos

### DISCUSSION THREAD

**Discussion by code-press** [vercel/vercel#5755] [Help] [answered]
I have my app hosted at GitHub, and am trying to import it (into my team account).  I select the "Add GitHub org or account" option.  pops the auth window and I got through auth, window closes.  but the page never adds my account or lets be browse the repos so I can add them.  Tried both safari and chrome, and multiple GitHub accounts to no avail. 

Any ideas?

**Comment by vindecodex** [comment, marked as answer]
@code-press Have you come up with this form and click install ?

![Screenshot from 2021-02-03 12-22-43](https://user-images.githubusercontent.com/50949760/106698019-ea3d4980-661a-11eb-9f59-58f542fd0c9d.png)

**Reply by clemsontiger** [reply]
I'm having the same issue. I installed vercel and gave it all permissions into my github but my github account name never shows on the "GIT SCOPE" when going to create a Git Repository. Any ideas?

**Reply by melladoma** [reply]
I had the same issue and upon ticketing, Vercel staff suggested me to leave a certain organization I was part of in my GitHub account and try importing again; that worked for me.

**Comment by Igwefran6** [comment]
Has anyone solved this?

**Comment by OivinF** [comment]
I'm seeing this exact issue and cannot find a fix

**Comment by eyesonnumber** [comment]
still facing same issue..what is this vercel, very frustrating..

**Comment by VeroniqueHuang** [comment]
The issue is still there, not able to see my project in the list.
The app is definetly installed "installed 8mn ago", even by deleting re-installing, still facing this issue.

**Comment by Ce1ling** [comment]
The issue is still there, and I'm 100% sure it's installed in my GitHub Apps:

![image](https://github.com/user-attachments/assets/10989622-bfbc-4265-8825-554b1c78efeb)

But the "Connected Git Repository" in Vercel just cannot be displayed,
It actually shows up after a while (maybe a few days later), but why is this happening?

**Comment by chrisweb** [comment]
I have only added permissions for one repository so far, I now wanted to create a new project and then add the permission for a second repository so that it can be connected to that new project

So I go my overview page, then I click on **Add New...** and select **Project**

This opens a new project page, but when I click on **Adjust GitHub App Permissions →**

![vercel_github_repository_problem_02](https://github.com/user-attachments/assets/cf16e3fc-6fb6-40d7-8ad5-8a8994626acc)

I get the following error:

![vercel_github_repository_problem_01](https://github.com/user-attachments/assets/bac955e5-9604-4e3c-ad38-1ba1bcc7d778)

I disabled my extensions and tried again, but I got the same error message again

For now, as a workaround, I went to this [https://github.com/settings/installations](https://github.com/settings/installations) page on GitHub, clicked on the **Configure** button (on the **Vercel** row), then I scroll down to the **Repository access** section, there **Only select repositories** is selected, below that is a **Select repositories** select box, I click the down arrow and then search for the new repository I want to add, then I click on the repository in the list of results, now that it got added to the list of repositories that Vercel will have access to I click on the save button to finilize the changes

**Reply by amyegan** [reply]
Hey @chrisweb. I'm not sure what's causing that error in your case, but you can manage settings for the Vercel app directly on GitHub. Go to account settings, select **Applications**, then click the **Configure** button next to the Vercel app. That will take you to the management page where you can change your repository access selections.
